site/docs/blog/namespace-enforcement.md

```diff
@@ -0,0 +1,7 @@
+# Namespace enforcement in Nessie
+
+Starting from Nessie version 0.52.3, it is required to have existing Namespaces before creating or committing to tables.
+<br>In case of tables that have implicit namespaces, new commits will fail after the upgrade as this rule will be enforced.
+<br>Therefore, it is necessary to explicitly create the namespaces using the using SQL command (for Spark, `CREATE NAMESPACE catalogName.namespaceName`) or Iceberg Catalog API.
```

Comment:
Is it not `CREATE DATABASE` in Spark?

Reply:
Please also mention that the `create-missing-namespaces` command of the Content Generator CLI.

Reply:
Both create databases and create namespaces can be used for same purpose. Not sure if it is because of Iceberg spark extensions. But both of them works.

<img width="457" alt="image" src="https://user-images.githubusercontent.com/5889404/236605879-400eb91e-02e6-4834-8ce9-3bea32def997.png">

Reply:
> Please also mention the create-missing-namespaces command of the Content Generator CLI (https://github.com/projectnessie/nessie/pull/6280).

ok

Reply:
Added about content generator tool and also had to update the tool's readme as it was outdated